src/archive.rs

```diff
@@ -25,6 +25,11 @@ impl CompressionMethod {
         .to_string()
     }
 
+    /// Lists compression methods
+    pub fn get_compression_methods() -> Vec<Self> {
+        vec![CompressionMethod::TarGz]
```

Comment:
Can't we use strum for this? Then we don't have to maintain this list.

Reply:
I was looking for a method for listing the possible variants of an enum but couldn't find one... I'll check

Reply:
I still think you should use https://github.com/Peternator7/strum#EnumIter for this.

Reply:
Or just remove it as we don't need it right now.

Reply:
Done :)